src/trusted/service_runtime/build.scons

```diff
@@ -21,6 +21,24 @@ if env.Bit('windows'):
 else:
   syscall_impl = 'posix/nacl_syscall_impl.c'
 
+if os.environ.get('TRACING_ENABLED') == '1':
+    env.Append(CCFLAGS=['-DTRACING=1'])
+    env.Append(CFLAGS=['-DTRACING=1'])
+    env.Append(CPPFLAGS=['-DTRACING=1'])
+    env.Append(CXXFLAGS=['-DTRACING=1'])
+
+#if os.environ.get('TRACING_TO_FILE') == '1':
+#    env.Append(CCFLAGS=['-DTRACING_TO_FILE=1'])
+#    env.Append(CFLAGS=['-DTRACING_TO_FILE=1'])
+#    env.Append(CPPFLAGS=['-DTRACING_TO_FILE=1'])
+#    env.Append(CXXFLAGS=['-DTRACING_TO_FILE=1'])
```

Comment:
Just a code style suggestion: if this part is commented out and will never be used, we can delete it.